### PR TITLE
{Storage} `az storage account network-rule add`: add example for subnet usage

### LINKED ISSUE #13131: How to use when virtual network exists in a different resource group?
When using the `az storage account network-rule add` command, it's possible to specify the resource group name using the `--resource-group` argument. This argument however, seems to be used for discovering both the storage account and the virtual network. So both the storage account and the virtual network should exist in the same resource group. But how to use this command when that's not the case?

I've tried to specify the entire resourceId of the virtual network for the `--vnet-name` argument, but that doesn't seem to be valid solution. Maybe there's another way to add a network rule to the storage account? Maybe also good to extend the documentation on this page a bit to mention what the `--resource-group` argument refers to.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ac52f120-deaa-0a12-3663-8bf51cbbd234
* Version Independent ID: b7e2694c-c7be-8d1f-da00-fdd8b60f327e
* Content: [az storage account network-rule](https://docs.microsoft.com/en-us/cli/azure/storage/account/network-rule?view=azure-cli-latest#az-storage-account-network-rule-add)
* Content Source: [src/azure-cli/azure/cli/command_modules/storage/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_help.py)
* Service: **storage**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by Juliehzl**
**Description**  

Fix #13131 
Add one more example for subnet in another resource group.

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S170

File: src/azure-cli/azure/cli/command_modules/storage/_help.py [hunk 1]
```diff
@@ -278,7 +278,9 @@
   - name: Create a rule to allow a specific address-range.
     text: az storage account network-rule add -g myRg --account-name mystorageaccount --ip-address 23.45.1.0/24
   - name: Create a rule to allow access for a subnet.
-    text: az storage account network-rule add -g myRg --account-name mystorageaccount --vnet myvnet --subnet mysubnet
+    text: az storage account network-rule add -g myRg --account-name mystorageaccount --vnet-name myvnet --subnet mysubnet
+  - name: Create a rule to allow access for a subnet in another resource group.
+    text: az storage account network-rule add -g myRg --account-name mystorageaccount  --subnet $subnetId
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/storage/_help.py:283]: `$subnetId` It's not appropriate to include `$`, right?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/storage/_help.py:283]: It is placeholder. resource id is too long to put it in example.

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/storage/_help.py:283]: OK

**Pull request merged**